### ISSUE TITLE
I don't see option Firewalls and virtual networks in Task 2, Point 2. 

### ISSUE THREAD

**Issue by robert-dzikowski** [microsoft/azuredevopslabs#668]
I followed the tutorial until Task 2, point 2 but I don't see the 
![azure-sql-server](https://user-images.githubusercontent.com/12864860/236326114-c744b02e-72b2-44b1-8320-de1e46f2e128.png)
option Firewalls and virtual networks.

**Comment by shad-1** [comment]
@robert-dzikowski That's correct. The options in the Security section for this resource have changed. I'm planning to make an edit with an update on how to complete this step with private endpoints.
